guides/paper-iron-layout.md

```diff
@@ -0,0 +1,320 @@
+how you want to implement your layout, and then install only the elements
+that you need.
+
+```bash
+bower install -S PolymerElements/paper-header-panel
+bower install -S PolymerElements/paper-toolbar
+bower install -S PolymerElements/paper-drawer-panel
+bower install -S PolymerElements/paper-icon-button
+bower install -S PolymerElements/paper-tabs
+bower install -S PolymerElements/paper-tab
+bower install -S PolymerElements/paper-drawer-panel
+bower install -S PolymerElements/iron-icons
+bower install -S PolymerElements/iron-flex-layout
+```
+
+We'll assume that you can import these elements from `/bower_components/`.
+
+## Creating a header
+
+This section shows you how to:
+
+* create a standard layout with `paper-header-panel` and `paper-toolbar`,
+  which is probably the most common and easiest layout
+* use a custom element for a header 
+* add icons to a header
+* set the height of a header
+* add tabs to a header
+* modify the disply and behavior of a header
+
+### Creating a header with `paper-toolbar`
+
+The code below uses a `paper-header-panel` as the container of the
+page and a `paper-toolbar` as a header. When a `paper-toolbar` is a 
+child of `paper-header-panel`, the panel automatically displays 
+the toolbar as the header. All other 
+children are placed in the content area.
+
+```hmtl
+...
+<head>
+...
+  <link rel="import" 
+        href="/bower_components/paper-header-panel/paper-header-panel.html">
+  <link rel="import" 
+        href="/bower_components/paper-toolbar/paper-toolbar.html">
+  <link rel="import" 
+        href="/bower_components/iron-flex-layout/iron-flex-layout.html">
+...
+<body class="fullbleed vertical layout">
+  <!-- paper-header-panel must have an explicit height -->
+  <paper-header-panel class="flex">
+    <paper-toolbar>
+      <div>Header</div>
+    </paper-toolbar>
+    <div>Content</div>
+  </paper-header-panel>
+</body>
+...
+```
+
+[Demonstration](assets/header-and-toolbar.html)
+
+`paper-header-panel` **must have an explicit height**. See the list item
+on `flex` below for an explanation of why the code above works.
+
+`fullbleed`, `vertical`, `layout`, and `flex` are part of the 
+`iron-flex-layout` class. We use them in our examples as an easy way 
+to create a responsive design, but the `paper` elements do not depend 
+on them. Below is a description of each class used in the example above:
+ 
+* `fullbleed` instructs `body` to occupy the entire viewport. 
+* `vertical` and `layout` instruct `body` to stack elements 
+  vertically (use `vertical horizontal` to stack horizontally). `vertical`
+  must be accompanied by `vertical` or `horizontal`. It is meaningless
+  on its own.
+* `flex` instructs `paper-panel-header` to stretch to the entire 
+  size of its parent, in this case `body` (which is set to fill the entire 
+  viewport, hence achieving a responsive design).
+
+See [Flexbox layout with iron-flex-layout](/guides/flex-layout) for more
```

Comment:
is /guides/flex-layout a new guide that uses the classes or is it the old page from 0.5 that uses attributes?